public/chat/santaChat.js

```diff
@@ -81,4 +81,13 @@ $(async function() {
   $('.alert .btn-close').on('click', function() {
     $('.alert').hide();
   });
+
+  $('#deleteMessageDialog').on('click', () => {
```

Comment:
I would prefer this #deleteMessageDialog to be renamed to deleteMessageButton, as it is a button